moped-database/migrations/1651001224000_alter_project_list_view_funding/up.sql

```diff
@@ -0,0 +1,96 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with project_person_list_lookup AS (
+  SELECT
+    mpp.project_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ',') AS project_team_members
+   FROM 
+    public.moped_proj_personnel mpp
+        JOIN public.moped_users mu ON mpp.user_id = mu.user_id
+        JOIN public.moped_project_roles mpr ON mpp.role_id = mpr.project_role_id
+      WHERE (mpp.status_id = 1)
+      GROUP BY mpp.project_id
+),
+ funding_sources_lookup AS (
+  SELECT
+    mpf.project_id,
+    string_agg(mfs.funding_source_name, ', ') AS funding_source_name
+  FROM
+    moped_proj_funding mpf
+    LEFT JOIN moped_fund_sources mfs ON mpf.funding_source_id = mfs.funding_source_id
+    GROUP BY mpf.project_id
+)
+  SELECT 
+    mp.project_uuid,
+    mp.project_id,
+    mp.project_name,
+    mp.project_description,
+    mp.project_description_public,
+    mp.ecapris_subproject_id,
+    mp.project_order,
+    mp.current_status,
+    mp.timeline_id,
+    mp.current_phase,
+    mp.end_date,
+    mp.project_length,
+    mp.fiscal_year,
+    mp.capitally_funded,
+    mp.date_added,
+    mp.added_by,
+    mp.is_retired,
+    mp.milestone_id,
+    mp.task_order,
+    COALESCE(mp.status_id, 0) AS status_id,
+    mp.updated_at,
+    ppll.project_team_members,
+    me.entity_name AS project_sponsor,
+    string_agg(DISTINCT me2.entity_name, ', ') AS project_partner,
+        CASE
+            WHEN (mp.status_id = 0 OR mp.status_id IS NULL) THEN NULL::text
+            WHEN (mp.status_id = 1) THEN mp.current_phase
+            ELSE mp.current_status
+        END AS status_name,
```

Comment:
I think you may want to outdent this back to the same column that the fields in the `SELECT` clause lay in. The `WHEN`s and `ELSE` should be in from the `CASE` but the `CASE` is a field-like object.